.mbedignore

```diff
@@ -0,0 +1,32 @@
+*/test/*
+mbed-os/rtos/*
+mbed-os/features/FEATURE_LWIP/*
+mbed-os/features/unsupported/*
+mbed-os/features/cellular/*
+mbed-os/features/lorawan/*
+mbed-os/features/nanostack/*
+mbed-os/features/netsocket/*
+mbed-os/components/802.15.4_RF/*
+mbed-os/components/storage/blockdevice/COMPONENT_SD/util/*
+mbed-os/components/wifi/*
+mbed-os/features/frameworks/mbed-trace/*
+mbed-os/features/frameworks/mbed-client-cli/*
+mbed-os/features/frameworks/mbed-client-randlib/*
+mbed-os/features/frameworks/nanostack-libservice/source/*
+mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/*
+mbed-os/features/frameworks/mbed-coap/*
+mbed-os/features/lwipstack/*
+mbed-os/features/nfc/*
+mbed-os/platform/mbed_alloc_wrappers.c
+mbed-os/platform/mbed_retarget.c
+mbed-os/targets/TARGET_STM/TARGET_STM32WB/TARGET_STM32WB55xx/TOOLCHAIN_GCC_ARM/*
+mbed-os/usb/device/USBAudio/*
+mbed-os/drivers/I2CSlave.cpp
+mbed-os/storage/kvstore/source/kvstore_global_api.cpp
+fota/fota_source.c
+fota/fota_manifest_v1.c
+fota/fota_manifest_v3.c
+fota/fota.c
+fota/fota_app_ifs.c
+fota/fota_delta.c
+fota/fota_event_handler.c
```

Comment:
Why exactly is this file needed?

And why does it list `fota.c` which is coming in this PR?

Reply:
removed